rails/benchmarks/bm_activerecord_create.rb

```diff
@@ -4,10 +4,7 @@
 
 require_relative 'support/benchmark_rails.rb'
 
-ActiveRecord::Base.establish_connection(
-  adapter: 'sqlite3',
-  database: ':memory:'
-)
+ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
```

Comment:
I think it should work in Rails 3.2, but only if you don't pass any argument at all. I think accepting an URL was implemented only on Rails 4.